CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.0.0
+
+<!-- generated comparing v5.0.0-rc.1..next -->
+
+_Sep 16, 2021_
+
+After over 400 days of development and over 40 canary releases, we are excited to introduce [MUI Core v5.0.0](https://mui.com/blog/mui-core-v5/)!
+
+Some statistics with the released of v5.0.0 compared to the one of v4.0.0:
+
+- 5,832 new commits
+- From 2M downloads/month to 9.5M downloads/month on npm
+- From 350k users/month to 700k users/month on the documentation
+
+A big thanks to the 600+ contributors who made the release possible!
+
+The 5.0.0 version includes all changes done in the alpha, beta, and rc releases listed below.
+
```

Comment:
Group in the same paragraph, the two sentences are on the same topic: the changes included.

```suggestion
```